package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "jasmine-project",
+  "name": "code-challenges",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "dependencies": {
-    "jasmine": "^3.6.3"
+    "jest": "*"
```

Comment:
The version number here is not necessary and adds an additional step during upgrades.